lib/pdf_fill/forms/va21p527ez.rb

```diff
@@ -1711,7 +1711,7 @@ def to_checkbox_on_off(obj)
       end
 
       def to_radio_yes_no(obj)
-        obj ? 0 : 1
+        obj ? 1 : 2
```

Comment:
Same here with the to_i

Reply:
this function takes a boolean value and intentionally returns the int 1 if true, 2 if false. We don't want to return a string value in this function.